views/flathub_edit.tpl

```diff
@@ -1,5 +1,16 @@
 % rebase('base.tpl')
-
+<script>
+async function getDescription() {
+    url = '/flathub/description/{{app.flatpak_id}}';
+    response = await fetch(url);
+    values = await response.json();
```

Comment:
don't think you need `await` here

Reply:
It prevents the function from trying to write to the document before having received the data, since it is an async function. It shouldn't slow the rest of the application down.